samples/nrf9160/modem_shell/src/location/location_shell.c

```diff
@@ -589,7 +591,7 @@ int location_shell(const struct shell *shell, size_t argc, char **argv)
 	while ((opt = getopt_long(argc, argv, "m:t:", long_options, &long_index)) != -1) {
 		switch (opt) {
 		case LOCATION_SHELL_OPT_GNSS_TIMEOUT:
-			gnss_timeout = atoi(optarg);
+			gnss_timeout = atof(optarg);
```

Comment:
This works also for negative numbers, probably need to check that the value is positive?

Reply:
We haven't checked it in the past. Change to float doesn't change this behavior. I checked the negative value behavior and we are ignoring those silently in Location library. Whether that's good or not is another question but this PR is not changing that.

Reply:
Ok, that's fine then.